src/index.ts

```diff
@@ -23,8 +26,9 @@ log += '\n\n';
 log += '** Statistics **\n';
 log += lottery.getDurations();
 log += '\n\n';
-log += 'TOTAL DURATION: ' + lottery.totalDuration;
+log += `TOTAL DURATION ${lottery.totalDuration} IN MILLISECONDS.`
 
-var timestamp = Date.now();
-var filename = './results/lottery-result' + timestamp + '.txt';
-writeFile(filename, log, () => console.log('Check result file ' + filename));
+if (!fs.existsSync(dir)){
+    fs.mkdirSync(dir);
+}
```

Comment:
Nice addition, I didn't know how make this in Node. Again, this shows my lack of knowledge in this technology.